sdk/diffgram/file/view.py

```diff
@@ -42,20 +42,21 @@ def get_label_file_dict(self, schema_id = None, use_session = True):
 
     endpoint = "/api/v1/project/" + self.project_string_id + \
                "/labels/view/name_to_file_id"
-    params = {'schema_id': schema_id, 'directory_id': self.directory_id}
+    params = {'schema_id': schema_id}
     if use_session:
         response = self.session.get(self.host + endpoint, params = params)
     else:
         # Add Auth
         response = requests.get(self.host + endpoint,
                                 params = params,
-                                headers = {'directory_id': str(self.directory_id)},
+                                headers = {'schema_id': str(schema_id)},
```

Comment:
Thanks Anthony!
I'm wondering if we can remove the headers param too? It seems that we are already sending it in the query params. As seen here I feel there's nowhere we use the schema_id in the header: https://github.com/diffgram/diffgram/blob/29067ea49f4a8bd770ce8cd0b2220a0e5cd9420b/default/methods/annotation/labels/view.py#L36

And 2nd reason for moving away from custom headers is that we need to add custom ingresses rules to the nginx ingress in the k8s context like this: 

https://github.com/diffgram/diffgram-helm/blob/fdc0c50fae882ac2a9cfb3c72d5da34e85fe53d5/templates/ingress.yaml#L28

Reply:
Hey Pablo  That makes sense!  Agree 100%